tip-calculator/index.js

```diff
@@ -0,0 +1,14 @@
+function hackTip() {
+    var bill = document.getElementById("bill").value;
+    var rating = document.getElementById("rating").value;
+    var tip = document.getElementById("tip").value;
+    var damage = document.getElementById("damage").value;
+    var total = bill * rating;
+    var totalBill = parseFloat(bill) + parseFloat(total);
+    document.getElementById("tip").innerHTML = total;
+    document.getElementById("damage").innerHTML = totalBill;
+}
+
+document.getElementById("hack").onclick = function() {
```

Comment:
i think you should start using arrow functions

.onclick = () => hackTip()

Reply:
Yea, I do typically, but this was my first hackathon and it honestly threw me for a bit of a loop at first.